pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -72,6 +72,7 @@ default-package-overrides:
   # gi-gdkx11-4.x requires gtk-4.x, which is still under development and
   # not yet available in Nixpkgs
   - gi-gdkx11 < 4
+  - pandoc-crossref ==0.3.4.2
```

Comment:
Thanks for changing the location of this.

Could add a comment above this line with why it is added here, and when we can expect to be able to remove it?

Reply:
Done.